documents/papernach.md

```diff
@@ -116,7 +135,32 @@ $api->subscription->createSubscriptionRegistration(array('customer' => array('na
 ```
 
 **Parameters:**
-All parameters listed [here](https://razorpay.com/docs/api/recurring-payments/paper-nach/authorization-transaction/#121-create-a-registration-link) are supported
```

Comment:
change the parameter URL in readme  to - https://razorpay.com/docs/api/payments/recurring-payments/paper-nach/create-authorization-transaction/#112-create-an-order

Reply:
please update

Reply:
added all param link for `customer` & `subscription_registration` but not showing. can you check this link
https://github.com/razorpay/razorpay-php/blob/papernach_doc_correction/documents/papernach.md?plain=1#L136